jubako/src/main/java/com/justeat/jubako/ContentDescriptionCollection.kt

```diff
@@ -31,17 +31,10 @@ class ContentDescriptionCollection(var listener: Listener? = null) {
     fun clear() {
         val size = mContentDescriptions.size
         mContentDescriptions.clear()
+        mContentDescriptions = mutableListOf()
```

Comment:
As we `clear` above, why do we need a new instance ?

Reply:
its a good question it was to remove the association from the old list instead of working on the same cleared list since it was being consumed by an adapter, this whole collection business needs a rethink and not sure if I did this to solve a bug, will revisit later.